src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -330,6 +330,7 @@ public PEReader Resolve(string simpleName)
             }
         }
 
+#if NET
```

Comment:
I didn't understand what wrapping these methods in `#if NET` was accomplishing, could you please explain?